src/utils/preprints.js

```diff
@@ -54,7 +54,8 @@ export function getPdfUrl(preprint = {}) {
       encoding.contentUrl && encoding.encodingFormat === 'application/pdf'
   );
 
-  return encoding && encoding.contentUrl;
+  // doesn't seem to do anything, but FitH should set PDF view to Fit Horizontally
+  return encoding && (encoding.contentUrl+"?#view=FitH");
```

Comment:
Adobe's "Open" parameter [docs](https://www.adobe.com/content/dam/acom/en/devnet/acrobat/pdfs/pdf_open_parameters.pdf)

Reply:
`contentUrl` could already have some query string parameters. I would vote to remove that.
If you feel strongly about it, safely merging the query strings params would be better.

Reply:
Yep - reverted!  Resolving!